docroot/modules/custom/foia_webform/tests/src/Kernel/AgencyLookupServiceTest.php

```diff
@@ -17,41 +15,26 @@
  *
  * @package Drupal\Tests\foia_webform\Kernel
  */
-class AgencyLookupServiceTest extends KernelTestBase {
+class AgencyLookupServiceTest extends FoiaWebformKernelTestBase {
 
   /**
    * Modules to install.
    *
    * @var array
    */
   public static $modules = [
-    'system',
-    'user',
-    'webform',
-    'webform_template',
-    'foia_webform',
-    'field',
-    'node',
     'entity_reference',
-    'menu_ui',
-    'field_permissions',
-    'taxonomy',
     'filter',
-    'text',
+    'menu_ui',
   ];
 
   /**
    * {@inheritdoc}
    */
   protected function setUp() {
     parent::setUp();
-
     $this->installConfig(['system', 'taxonomy']);
     $this->installSchema('user', 'users_data');
-    $this->installEntitySchema('node');
-    $this->installEntitySchema('user');
-    $this->installEntitySchema('taxonomy_term');
-
   }
```

Comment:
I think we do a lot of things manually in this test that we do in the `FoiaWebformApiKernelTestBase`:
1. Setup a mock webform (we setup a real webform in a base class that we could probably use)
2. Add the agency component content type (we do that in `setupAgencyComponent`, so maybe it can be moved to `FoiaWebformKernelTestBase`)
3. Set up `field_request_submission_form` (handled as part of `setupAgencyComponent`)
4. Create the agency vocabulary and add an agency taxonomy term (we do that in the `setup` method)
5. Create a test agency component that is associated to a webform and an agency (we do that as part of `setupAgencyComponent`)